docs/src/references/tedge-cumulocity-proxy.md

```diff
@@ -11,9 +11,11 @@ thin-edge device.
 It automatically handles authenticating with a JWT, avoiding the need for clients to support MQTT to retrieve this
 information.
 
-The API can be accessed at `http://{ip}:8001/c8y/{c8y-endpoint}`. `ip` is configured using the `c8y.proxy.bind.address`
-configuration
-(the default value is `127.0.0.1`) and the port can be changed by setting `c8y.proxy.bind.port`.
+The Cumulocity HTTP API can be accessed at `http://{host}:{port}/c8y/{c8y-endpoint}`. Configuration settings
+`c8y.proxy.client.host` and `c8y.proxy.client.port` are used to configure `{host}` and `{port}` parts of the base URL
+which will be used by thin-edge components to make requests to the C8y Proxy. `c8y.proxy.bind.address` and
```

Comment:
If my understanding is correct, this should stress that the client settings configure the mapper and not the agent (which is what I, as a user, would assume the client setting changes as it is how the equivalent MQTT settings work). I guess the main clue as to this being the case without knowing the implementation details is that the `c8y` settings only affect the mapper as that's the only component that cares about Cumulocity.

Reply:
It is defining how clients can reach the endpoint.

The mapper is the only component that uses the setting directly, however the value of the setting will be used within command payloads which are consumed by other clients (e.g. custom device connectors)

Reply:
I get that, my point was "there isn't any point setting this on a child device as the value won't be used", which is unlike what happens for the mqtt client settings. It probably doesn't have much of an impact, but the logic behind why it's like that doesn't seem like a thing that's necessarily obvious to a user so might be worth drawing attention to.

Reply:
I agree with @jarhodes314. We must be explicit telling that both `c8y.proxy.bind.address` and `c8y.proxy.client.host` have to be set on the box running the `c8y` mapper. The latter will be forwarded by the c8y mapper to its clients. `c8y.proxy.client.host` can also be directly set on the clients which aim to use the c8y REST API.

Reply:
My bad, forgot about this ticket. Description tweaked in 0159f91, is it OK now?